### ISSUE TITLE
[FEATURE] Use favicon for listen to the radio from website

### ISSUE THREAD

**Issue by tommylung** [TheBoredTeam/boring.notch#159]
There is no thumbnail image when I listen to the radio from the browser. Perhaps we could consider using the favicon when no image is available from the website. Thank you!

<img width="574" alt="Screenshot 2024-10-23 at 12 14 16 PM" src="https://github.com/user-attachments/assets/798b1358-f366-4f7f-8c0d-b365ea2b462b">

**Checks**
- [x] I haven't found any duplicates with my issue.

**Comment by danieltaylor** [comment]
Agreed that this would be good.  May be better to rely on apple-touch-icon or something similar (since it will likely be higher resolution, if available) with favicon as a fallback.

**Comment by UltraRouge** [comment]
Yeah. Video files also don't have an icon, but they could use the app icon (QuickTime Player or VLC, for example)
